modules/bbp-experiment/src/main/resources/schemas/bbp/experiment/patchedcell/v0.1.0.json

```diff
@@ -138,31 +138,24 @@
         }
       ]},
     {
-      "@id" : "this:AnalogToDigitalConverterShape",
-      "@type" : "sh:NodeShape",
-      "property" : [
-        {
-          "path" : "nsg:analogToDigitalConverter",
-          "name" : "Analog to digital converter",
-          "node":"this:ConverterShape"
-        }
-      ]
-    },
-    {
       "@id" : "this:DigitalToAnalogConverterShape",
```

Comment:
I would suggest to extends the ConverterShape here instead of having a property nsg:digitalToAnalogConverterShape
```json
"and": [
         {
          "node":"this:ConverterShape"
        },
         {
           "property":[...]
          }
]
```

Reply:
I have written it in the way you've suggested.